docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -220,12 +221,15 @@ Json Parameters:
       container
 -   **ExposedPorts** - An object mapping ports to an empty object in the form of:
       `"ExposedPorts": { "<port>/<tcp|udp>: {}" }`
+-   **VolumeDriver** - A string value containing the volume driver to use, `local` by default.
```

Comment:
@calavera My point was about clarifying the relationship between "plugins" and "drivers" because that isn't described in the docs anywhere else from a user's point of view. Can you add this sentence to the API docs or do you want me to do a PR?